src/node.cc

```diff
@@ -4213,6 +4213,17 @@ inline void PlatformInit() {
     } while (min + 1 < max);
   }
 #endif  // __POSIX__
+#ifdef _WIN32
+  for (int fd = 0; fd <= 2; ++fd) {
+    auto handle = reinterpret_cast<HANDLE>(_get_osfhandle(fd));
+    if (handle == INVALID_HANDLE_VALUE ||
+        GetFileType(handle) == FILE_TYPE_UNKNOWN) {
+      _close(fd);
```

Comment:
Win2008 and 2012 fail with errno `EBADF`. Win 10 happily closes the handle. But regardless of that, `_open` works as expected on all Win versions.